usecases/large-scale-processing/index.md

```diff
@@ -0,0 +1,106 @@
+while for more demanding cases, a 20km grid is better. In our example, we choose to work with 20km tiles because the workflow was quite demanding. A smaller 
+tile size can also result in less unneeded processing when your target area has an irregular shape, like most countries and continents.
+
+A couple of basic grids can be found here:
+https://artifactory.vgt.vito.be/webapp/#/artifacts/browse/tree/General/auxdata-public/grids
+
+A grid can be masked based on the countries we want to load, the following script shows an example:
+
+```python
+import geopandas as gpd
+europe = gpd.read_file(gpd.datasets.get_path("naturalearth_lowres"))
+europe = europe[europe.continent=="Europe"]
+countries = europe[europe.name.isin(EU27)]
+df = gpd.read_file("https://artifactory.vgt.vito.be/auxdata-public/grids/LAEA-20km.gpkg",mask=countries)
+```
+
+## Prepare job attributes
+
+Next to the tiling grid, we recommend to also determine other properties required by your processing jobs up front. This allows you to properly
+review those properties before starting to process. Examples include simple things like a job title or tile specific processing parameters, but also an 
+attribute to determine processing order.
+
+In this step, you may also want to make sure to already determine the correct tile extent in the coordinate system of your tile grid. 
+Providing exact coordinates in the right projection is necessary to ensure pixel-perfect alignment of your tiles.
+
+## Tuning your processing job
+
+Before kicking off large processing, you want to be very sure that the correct output is generated, and that you have sufficient credits and resources
+available to finish your job in time. This can be done by simply running various jobs, and using the statistics reported in the metadata to determine average 
+parameters.
+
+For instance, for the case of processing the EU27 croptype map, consisting of ~11000 20km tiles, we made the following calculations op front:
+
+- Average runtime was 30 minutes, which means that it would take ~15 days of continous processing with 15 parallel jobs.
+- Average cost was below 100 credits, so we would be able to process with a budget of 1100000 credits.
+
+To achieve these numbers, we did have to optimize batch job settings and also the overall workflow to reduce resource usage!
+
+A common bottleneck to parallellization is the memory consumption, and it can be useful to know the maximum memory allocation on a single machine in
+your backend of choice. For instance, in a cloud environment with 16GB per machine and 4 cpu's, using slightly less than 4GB per worker is efficient as you can fit 4 parallel workers on a single VM, while requiring 6GB would fit only 2 workers and leave about 4GB unused.
+
+In our example, we used the Geotrellis backends, which has [these tunables](../../federation/#customizing-batch-job-resources-on-terrascope).
```

Comment:
Im love the term tunables, although I guess it doesn't exist. Maybe "execution options" or somethign similar is easier to understand?

Anyway, my main point here is that the link should be different though:

```suggestion
In our example, we used the Geotrellis backends, which has [these execution options](../../federation/index.md#customizing-batch-job-resources-on-terrascope).
```